.github/workflows/nightly-builds.yml

```diff
@@ -88,6 +88,7 @@ jobs:
           # leaving out some combinations (note that `build-test-prValidations.yml` doesn't run all test
           - { scalaVersion: "2.13", jdkVersion: "1.11.0", jvmName: "temurin:1.11.0", extraOpts: '' }
           - { scalaVersion: "2.13", jdkVersion: "1.17.0", jvmName: "temurin:1.17.0", extraOpts: '' }
+          - { scalaVersion: "2.13", jdkVersion: "1.21.0", jvmName: "temurin:1.21", extraOpts: '' }
```

Comment:
what about scala3 with jdk 21?

Reply:
btw, PR validation is still using jdk11. 

Maybe time to move to jdk17 and keep jdk11 only in the nightly build?

Reply:
I think it is good to keep the oldest as default because then we catch if a PR uses new JDK (and Scala) stdlib issues directly instead of not until the nightlies.

Not sure if it is worth it with one more build for JDK 21 or not.